src/app/medInria/resources/medInria.qrc

```diff
@@ -120,6 +120,21 @@
         <file>icons/user_add.png</file>
         <file>icons/user_edit.png</file>
         <file>medInria.qss</file>
+        <file>pixmaps/unknown.jpg</file>
+        <file>pixmaps/checkbox.png</file>
+        <file>pixmaps/checkbox-checked.png</file>
+        <file>pixmaps/checkbox-partial.png</file>
+        <file>pixmaps/radio.png</file>
+        <file>pixmaps/radio-checked.png</file>
+        <file>pixmaps/radio-partial.png</file>
+        <file>pixmaps/drop-down.png</file>
+        <file>pixmaps/treeview-branch-closed.png</file>
+        <file>pixmaps/treeview-branch-open.png</file>
+        <file>pixmaps/database_navigator_item_checker_checked.png</file>
+        <file>pixmaps/database_navigator_item_checker_unchecked.png</file>
+        <file>pixmaps/medInria-logo-homepage.png</file>
+        <!-- <file>pixmaps/medInria-splash.png</file> -->
```

Comment:
there are some commented lines left

Reply:
I would like to keep them to check and clean up in the near future.
It's not a forget